packages/tools/src/PnpmTool.ts

```diff
@@ -1,10 +1,9 @@
 import path from "path";
-import readYamlFile, { sync as readYamlFileSync } from "read-yaml-file";
+import yaml from "js-yaml";
```

Comment:
`read-yaml-file` - what was its supported node engines' range (for the specific version that we were using here)? and what is the range of the new package?

Reply:
`read-yaml-file` pulled in `js-yaml` as a dependency, so we were using `js-yaml` before as well. I checked and we're using v4 of `js-yaml` both before and after this PR so the supported version ranges shouldn't change at all